### PR TITLE
Add integration with Anitya for auto-flagging of outdated packages

### PR DESCRIPTION
**Pull request by jirutka**
~~It’s still work in progress, but anitya-import is already functional. 😸~~

It needs Lua modules [gversion](https://github.com/jirutka/gversion.lua) and [lzmq](https://github.com/zeromq/lzmq). I’ll create Alpine packages for them.

~~Depends on #16.~~

/cc @ncopa

### CODE REVIEW

**Comment by clandmeter** [comment]
Nice!

cc @fcolista, isn't that something you were looking into?

**Comment by fcolista** [comment]
Correct! Nice job Jirutka..thanks a lot!

File: tools/import.lua [hunk 1]
```diff
@@ -155,7 +155,7 @@ function import:getLocalRepoVersion(branch, repo, arch)
         where branch = ? and repo = ? and arch = ? ]]
     local stmt = self.db:prepare(sql)
     stmt:bind_values(branch, repo, arch)
-    local r = (stmt:step()==sqlite3.ROW) and stmt:get_value(0) or false
+    local r = (stmt:step() == sqlite.ROW) and stmt:get_value(0) or false
```
Comment (clandmeter) [inline tools/import.lua:159]: http://lua.sqlite.org/index.cgi/doc/tip/doc/lsqlite3.wiki#stmt_step

This mentions:

sqlite3.ROW: this is returned each time a new row of data is ready for processing by the caller. The values may be accessed using the column access functions. stmt:step() can be called again to retrieve the next row of data.

Comment (jirutka) [reply tools/import.lua:159]: The problem here is that `sqlite3` is not defined, your variable is named `sqlite`.

[tools/import.lua:1](https://github.com/clandmeter/aports-turbo/blob/9946cf26f67157f88bced4273bd6124ac7f22bf7/tools/import.lua#L1):

``` lua
local sqlite    = require("lsqlite3")
```

Comment (jirutka) [reply tools/import.lua:159]: BTW please review commits before “Add integration with Anitya [WIP]” in #16, it should not be in this PR. But #16 is not merged yet and I need it for this PR, that’s why it contains even commits from #16.

**Comment by jirutka** [comment]
I’m done. 🎉

Please read documentation in header of [tools/anitya-check-all.lua](https://github.com/jirutka/aports-turbo/blob/395d8f447d1cee942c66aff3b5cce9bf23d3a27a/tools/anitya-check-all.lua#L2-L17) and [tools/anitya-watch.lua](https://github.com/jirutka/aports-turbo/blob/395d8f447d1cee942c66aff3b5cce9bf23d3a27a/tools/anitya-watch.lua#L2-L17). There’s an explanation what does it actually do and how does it work. Feel free to ask if something is not clear.

Additional dependencies are already in Aports:
- [lua-gversion](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/lua-gversion) – my Lua module for parsing and comparing Gentoo-style versioning format that apk uses.
- [lua-lzmq](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/lua-lzmq) – bindings for zeromq (needed for anitya-watch, to connect to fedmsg).

**Pull request merged**